lp-collaborative-workspace/lp-cw-bridge/lp-cw-bridge-transformer/resources/transformation/ado2xwiki.atl

```diff
@@ -32,52 +46,54 @@ helper def : getConnectorSourceId(connector: ADOXX!CONNECTORType) : String =
 	connector
 	.refImmediateComposite()
 	.iNSTANCE
-	->select(
-		i |
-		(connector.fROM.class = i.class) and (connector.fROM.instance = i.name)
-	)
+	->select(i | (connector.fROM.class = i.class) and (connector.fROM.instance = i.name))
 	.flatten()
 	.first()
 	.id;
+
 -- Get the target[TO] instance id for the connector
 helper def : getConnectorTargetId(connector: ADOXX!CONNECTORType) : String = 
 	connector
 	.refImmediateComposite()
 	.iNSTANCE
-	->select(
-		i |
-		(connector.tO.class = i.class) and (connector.tO.instance = i.name)
-	)
+	->select(i | (connector.tO.class = i.class) and (connector.tO.instance = i.name))
 	.flatten()
 	.first()
 	.id;
+
 helper def : linkClassNumberMap : Map(String, Integer) =
-	ADOXX!INSTANCEType
-	.allInstances()
-	->iterate(
-		i;
-		accumulator : Map(String, Integer) = Map{} |
-		accumulator.including(i.id, 0)
-	);
+	ADOXX!INSTANCEType.allInstances()
+	->iterate(i; accumulator : Map(String, Integer) = Map{} | accumulator.including(i.id, 0));
 
 helper def : incLinkClassNumber(id: String) : Map(String, Integer) =
-	thisModule.linkClassNumberMap
-	.including(id, thisModule.linkClassNumberMap.get(id) + 1);
-
+	if (thisModule.linkClassNumberMap.get(id).oclIsUndefined()) then 
+		Map{('brokenLink', 0)}
```

Comment:
I've saw it in my previous review but didn't comment it.  I wonder if the value `brokenLink` is a good idea.  Usually, you would use some empty value in this kind of use case like `''` or for ATL, maybe OCLUndefined could work too (not sure about this one though).